### PR TITLE
[servo] fix image build by setting HARFBUZZ_SYS_NO_PKG_CONFIG=1

### PR DESCRIPTION
**Pull request by jankeromnes**
Fixes https://github.com/JanitorTechnology/janitor/issues/345

### CODE REVIEW

**Comment by jankeromnes** [comment]
@jdm could you please take a look at this fix?

Also, CI run should tell us if this actually fixes the Servo image build.

**Comment by jdm** [comment]
I'm not sure about the rules of when to use export or not, but that looks ok otherwise.

**Comment by jankeromnes** [comment]
Huh, I've never seen this build fail before:

```
Step 5/52 : RUN add-apt-repository ppa:git-core/ppa
 ---> Running in aeeb7f63f2b4
 The most current stable version of Git for Ubuntu.

For release candidates, go to https://launchpad.net/~git-core/+archive/candidate .
 More info: https://launchpad.net/~git-core/+archive/ubuntu/ppa
gpg: keyring `/tmp/tmpmkwcxqm1/secring.gpg' created
gpg: keyring `/tmp/tmpmkwcxqm1/pubring.gpg' created
gpg: requesting key E1DF1F24 from hkp server keyserver.ubuntu.com
gpgkeys: key E1DD270288B4E6030699E45FA1715D88E1DF1F24 can't be retrieved
gpg: no valid OpenPGP data found.
gpg: Total number processed: 0
gpg: keyserver communications error: keyserver helper general error
gpg: keyserver communications error: unknown pubkey algorithm
gpg: keyserver receive failed: unknown pubkey algorithm
Failed to add key.
The command '/bin/sh -c add-apt-repository ppa:git-core/ppa' returned a non-zero code: 1
Exited with code 1
```

(Totally unrelated to Servo, re-triggering again.)

File: servo/servo.dockerfile [hunk 1]
```diff
@@ -53,6 +53,11 @@ WORKDIR servo
 # Configure the IDEs to use Servo's source directory as workspace.
 ENV WORKSPACE /home/user/servo/
 
+# Work around a Servo build problem.
+RUN echo "\n# Work around https://github.com/servo/servo/issues/20712." >> /home/user/.bashrc \
+ && echo "HARFBUZZ_SYS_NO_PKG_CONFIG=1" >> /home/user/.bashrc
```
Comment (notriddle) [inline servo/servo.dockerfile]: This should be `export HAFBUZZ_SYS_NO_PKG_CONFIG=1`

The version without export won't be visible to sub-processes of the shell, like cargo.

**Comment by jankeromnes** [comment]
The fix worked!

servo-fixed

File: servo/servo.dockerfile (hunk 1 above)
Comment (jankeromnes) [reply servo/servo.dockerfile]: Fair enough, thanks for catching this!

Interestingly, we're not consistent with this throughout this repository, as shown by [searching for "PATH"](https://github.com/JanitorTechnology/dockerfiles/search?q=PATH).

**Pull request merged**